.github/workflows/release-alpha.yaml

```diff
@@ -4,7 +4,7 @@ on:
 jobs:
   # Release alpha version on NPM for Hive libraries
   npm:
-    uses: the-guild-org/shared-config/.github/workflows/release-snapshot.yml@main
+    uses: the-guild-org/shared-config/.github/workflows/release-snapshot.yml@feat-export-outputs
```

Comment:
See https://github.com/the-guild-org/shared-config/pull/534, required for the `extract-cli-version` step.